### DISCUSSION TITLE
Prop Ref in Pie/ResponsivePieCanvas is undefined

### DISCUSSION THREAD

**Discussion by Dlina1502** [plouc/nivo#2340] [Q&A]
ref to Canvas element is undefined in Pie/ResponsivePieCanvas charts.
But it works perfectly fine in the ResponsiveBarCanvas and returns a reference to the Canvas.

**Comment by jamesarosen** [comment]
This is because `ResponsiveBar`, `ResponsiveLine`, and `ResponsivePie` all use `ResponsiveWrapper`, which doesn't forward the `ref` passed in. l opened https://github.com/plouc/nivo/issues/2433 for this issue.
